llvm/test/Transforms/SCCP/conditions-ranges-with-undef.ll

```diff
@@ -46,8 +46,7 @@ define void @val_undef_range() {
 ; CHECK-NEXT:    br i1 [[BC_1]], label [[TRUE:%.*]], label [[FALSE:%.*]]
```

Comment:
The comment above this function is now outdated. I believe the transform is correct because the branch excludes undef.